default.nix

```diff
@@ -175,15 +178,16 @@ let
 
             echo "Starting napalm registry"
 
-            napalm-registry --snapshot ${snapshot} &
+            registrySock="$TMPDIR/registry.sock"
```

Comment:
```suggestion
            registrySock="$(mktemp -d)/registry.sock"
```

Reply:
Why nest this into another directory? We already set this up accordingly in the [builder sandbox](https://github.com/NixOS/nix/blob/fda835b231e4ae2cbcedf7bb6124443fa07c9ab4/src/libstore/build/derivation-goal.cc#L1818) and *if* this would interfere with build-specific files, the `mktemp` variant would have the same issue.